package com.cedaniel200.poo.talleres.excepciones.runtimeexception;

public class Division {
    public int dividir(int dividendo, int divisor){
        return dividendo / divisor;
    }
}
